modules/basics/src/main/java/com/opengamma/strata/basics/date/GlobalHolidayCalendars.java

```diff
@@ -845,8 +845,12 @@ private static void newZealand(List<LocalDate> holidays, int year) {
     } else {
       holidays.add(date(year, 4, 25));
     }
-    // queens birthday
+    // queen's birthday
     holidays.add(first(year, 6).with(firstInMonth(MONDAY)));
+    // queen's funeral
+    if (year == 2022) {
+      holidays.add(date(year, 9, 26));
+    }
```

Comment:
we seem to only be testing the NZ calendars until 2018 - should we add 2022 test data too because of this?